tests/test_modeling_tf_common.py

```diff
@@ -98,6 +99,14 @@ def _prepare_for_class(self, inputs_dict, model_class, return_labels=False) -> d
                 inputs_dict["labels"] = tf.zeros(self.model_tester.batch_size, dtype=tf.int32)
             elif model_class in TF_MODEL_FOR_NEXT_SENTENCE_PREDICTION_MAPPING.values():
                 inputs_dict["next_sentence_label"] = tf.zeros(self.model_tester.batch_size, dtype=tf.int32)
+            elif model_class in TF_MODEL_FOR_PRETRAINING_MAPPING.values():
+                # Only BERT needs the next sentence label for pre-training
+                if model_class.base_model_prefix in ["bert"]:
+                    inputs_dict["next_sentence_label"] = tf.zeros(self.model_tester.batch_size, dtype=tf.int32)
+
+                inputs_dict["labels"] = tf.zeros(
+                    (self.model_tester.batch_size, self.model_tester.seq_length), dtype=tf.int32
+                )
```

Comment:
If it's BERT specific, I'd prefer to override this in `TFBertModelTest` instead of putting model-specific stuff in a model-agnostic file

Reply:
How would you do that? As this is only for BERT + PRetraining.

Reply:
@LysandreJik What do you think of this approach to solve this?

Reply:
No, this isn't what I had in mind. I think overriding this in `TFBertModelTest` would be better. Similarly to how the `test_saved_model_with_hidden_states_output` is overriden in `TFLxmertModelTest`: redefining the test in the tester when it diverges to the common test.

We should always extrapolate that a model-specific change will lead to other model-specific changes down the road, and this would become unmaintainable. Overriding in the model-specific tester makes much more sense from a maintainability point of view.

Reply:
Ok got it! Sorry I wasn't really how to overwrite a test.

Reply:
Just made an update. Does-it looks better?